octoprint_flashforge/__init__.py

```diff
@@ -63,6 +65,13 @@ def __init__(self):
 		self._feature_settings = {
 			'autoUppercaseBlacklist': ['M146']		# LED control requires lowercase r,g,b
 		}
+
+		for setting in self._conn_settings.keys():
```

Comment:
I admit I have not looked at the changes in OctoPrint settings - does this replace the need for setting the default_settings? eg:
`default_settings["serial"] = dict_merge(default_settings["serial"], self._conn_settings)`
or is it doing something else?